docs/how-to/deploy_charmed_magma_orchestrator.md

```diff
@@ -18,9 +18,18 @@ Create an `overlay.yaml` file that contains the following content:
 
 ```yaml title="overlay.yaml"
 applications:
+  fluentd:
+    options:
+    domain: <your domain name>
+    elasticsearch-url: <your elasticsearch https url>
+    fluentd-chunk-limit-size: "2M"
```

Comment:
Do we need `fluentd-chunk-limit-size` and `fluentd-queue-limit-length` here?

Reply:
I'd keep the content here to a minimum

Reply:
Removed.